src/StaticAnalysisResultsInterchangeFormat/Converters/CppCheckError.cs

```diff
@@ -109,16 +109,16 @@ public static CppCheckError Parse(XmlReader reader, CppCheckStrings strings)
             }
         }
 
-        /// <summary>Converts this instance to <see cref="Issue"/>.</summary>
-        /// <returns>This instance as an <see cref="Issue"/>.</returns>
-        public Issue ToSarifIssue()
+        /// <summary>Converts this instance to <see cref="Result"/>.</summary>
+        /// <returns>This instance as an <see cref="Result"/>.</returns>
+        public Result ToSarifIssue()
         {
             if (this.Locations.Length == 0)
             {
-                throw new InvalidOperationException("At least one location must be present in a Unified Issue Store issue.");
+                throw new InvalidOperationException("At least one location must be present in a Unified Result Store result.");
```

Comment:
"Unified Issue Store" was an MS-centric name. Change "in a Unified Result Store Result" to just "in a Result". Look for other occurrences of "OES", "1ES", "Unified Issue Store", and "UIS".